discovery-provider/src/queries/get_health_test.py

```diff
@@ -13,15 +13,25 @@
 from src.queries.get_health import get_health
 
 
-def test_get_health(web3_mock, redis_mock, db_mock):
-    """Tests that the health check returns db data"""
+def test_get_health(web3_mock, redis_mock, db_mock, get_monitors_mock):
```

Comment:
why change this in this PR?

Reply:
Yeah I thought about separating this out. We generally want to keep changes likes this for different services separate right?

Reply:
oh we can have pr's with different services, i just wasn't sure why this change was made

Reply:
Got the health checks confused, changing this!